tests/phpunit/FunctionalRowTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace Keboola\DbWriter\Snowflake\Tests;
+
+use Keboola\Csv\CsvFile;
+use Keboola\DbWriter\Snowflake\Test\S3Loader;
+use Keboola\DbWriter\Writer\Snowflake;
+use Keboola\StorageApi\Client;
+use Symfony\Component\Process\Process;
+
+class FunctionalRowTest extends BaseTest
```

Comment:
there seems to be a lot of duplication with `FunctionalTest.php`.  How about making this class extend FunctionalTest?

Reply:
you're right. done :)